### ISSUE TITLE
From 1.0 to 1.5

### ISSUE THREAD

**Issue by saxrub** [manifestinteractive/jqvmap#223]
Hello
I just installed version 1.5 .
With my previous version 1.0 , the 
http://www.wnat.fr/visiolittoral.france.js map and 
http://www.wnat.fr/visiolittoral.colorsFrance.js worked well but with 1.5 , it causes JavaScript errors, and the MouseHover onRegionClick no longer work.
One idea would be welcome.
Regards
Pascal

**Comment by manifestinteractive** [comment]
@saxrub Can you please paste the your code for initializing the map?  I will need to see exactly how you are loading JQVmap on your site in order to duplicate this for debugging.

**Comment by manifestinteractive** [comment]
@saxrub After looking at that map file, it is pretty clearly malformed.  There are embedded markup code in your file that should not belong there.  That means that actual http://www.wnat.fr/visiolittoral.france.js file itself should not work.  The map files should ONLY contain PATH related data that looks like numbers, as those numbers are node points for generating SVG files.  Having embedded markup within that will cause weird things to happen.

You are likely not getting any errors on v1.0 because malformed map files like this just ignored the errors rather than reporting them.

Bottom line, your map file should not actually have worked at all.  The fact it did in v1.0 was actually a fluke.

**Comment by manifestinteractive** [comment]
FYI, if you wanted to try to fix you map file, you should look into the errors it was generating.  That was actually what clued me into the malformed map file:

```
jquery.vmap.js:1082 Error: Invalid value for <path> attribute d="M361.9,161.9l-2.9,1.9l3.4,4l5.9-0.1l-2-2.1v-0.1l-0.8-3L361.9,161.9z\u003Cline fill=none stroke=#CCCCCC stroke-width=0.3 stroke-linecap=round stroke-linejoin=round x1=362.4 y1=167.8 x2=361.9 y2=172.4"VectorCanvas.createPath @ jquery.vmap.js:1082JQVMap @ jquery.vmap.js:142jQuery.fn.vectorMap @ jquery.vmap.js:376(anonymous function) @ france.html:17j @ jquery-1.11.3.min.js:2k.fireWith @ jquery-1.11.3.min.js:2m.extend.ready @ jquery-1.11.3.min.js:2J @ jquery-1.11.3.min.js:2
jquery.vmap.js:1082 Error: Invalid value for <path> attribute d="stroke=#CCCCCCl-3.7-1.3l-2.8,1.7c-2.2-0.8-4.2-1.7-6.4-2.5l-2.9-2.3l-0.4-6.8l-3.5-0.6l-7.3,1l-10.6,8.5v3.7l-10.1-1.3l-3.7-5"VectorCanvas.createPath @ jquery.vmap.js:1082JQVMap @ jquery.vmap.js:142jQuery.fn.vectorMap @ jquery.vmap.js:376(anonymous function) @ france.html:17j @ jquery-1.11.3.min.js:2k.fireWith @ jquery-1.11.3.min.js:2m.extend.ready @ jquery-1.11.3.min.js:2J @ jquery-1.11.3.min.js:2
jquery.vmap.js:1082 Error: Invalid value for <path> attribute d="stroke=#CCCCCCc-0.8,0.5-1.6,1.1-2.3,1.6c-1,0.7-2,1.4-3,2.2l-2.9-1.3l-3.7,1.5l-4,2.9h-0.1"VectorCanvas.createPath @ jquery.vmap.js:1082JQVMap @ jquery.vmap.js:142jQuery.fn.vectorMap @ jquery.vmap.js:376(anonymous function) @ france.html:17j @ jquery-1.11.3.min.js:2k.fireWith @ jquery-1.11.3.min.js:2m.extend.ready @ jquery-1.11.3.min.js:2J @ jquery-1.11.3.min.js:2
jquery.vmap.js:1082 Error: Invalid value for <path> attribute d="M737.4,111.1l-5.2-4.5l-7.5-4.3l-8.1-1.5l-2.3-3.9v-6.6l-2.9-5.1l-4.5-5.4l-5.9,8.3l0.7,4.4l3.7,2.3l1.5,3.7l-3.6,3l-4.6,1.5l-0.8,5l-5,0.9l-3.1-5.9l-8.2-4.5l-7.4-0.7l-2.9,6.6v6.7l3.6,8.9v11.2l3.8,5.9l3.6,8.2l3-0.7l12.6-8.2v-13.5l-2.9-2.1v-8.3c1.3-0.7,2.5-1.4,3.7-2.2l3.9,5.2l5.7,1.5l15.7-5.2l7.5,0.7L737.4,111.1zM738,146.8l-4.4-5.4l-5.2-2.9l-7.5,8.3l2.3,6.5l3.6,0.7l3.8,1.6l5.9-5.2L738,146.8z\u003C/g\u003E\u003C/g\u003E\u003Cg\u003E\u003Cpolyline fill=none stroke=#8CA2C0 stroke-width=2.4 stroke-linecap=round stroke-linejoin=round points=737.5,147 733.1,141.8 728,138.8 720.5,147 722.8,153.6 726.4,154.3 730.1,155.8 736.1,150.8 737.5,147 "
```

If you look at the lines it mentions, you will see stuff like `<line fill=none stroke=#CCCCCC stroke-width=0.3` etc which is actually SVG markup.  This stuff should not existing in the file.  I tried cleaning this out to see if it fixed the problem, but it looks like a few map points are just complete junk and also cause issues.

Other than just recreating the map from scratch, which I cannot do, I am not sure how to fix that custom map file for you.

**Comment by saxrub** [comment]
Hi
Here an url to test V1 http://wnat.fr/france_map.1.0.html
Do you know a tool to clean an svg file produce by Adobe Illustrator ?

**Comment by saxrub** [comment]
I found !
When the error , the desired code was converted to lowercase by vmap .
So I change my codes in france.js file, modified a little piece of code and now it works!
You can try http://wnat.fr/france_map.1.5.html
